draft-ietf-quic-transport.md

```diff
@@ -730,20 +725,18 @@ location in all packet headers, making it straightforward for middleboxes, such
 as load balancers, to locate and use it.
 
 The client MUST choose a random connection ID and use it in Client Initial
-packets ({{packet-client-initial}}).  If the client has received any packet from
-the server, it uses the connection ID it received from the server.
-
-When the server receives a Client Initial packet, it chooses a new value for the
-connection ID and sends that in its response.  The server MUST send a new
-connection ID in any packet that is sent in response to a Client Initial packet.
-This includes Version Negotiation ({{packet-version}}), Server Stateless Retry
-({{packet-server-stateless}}), and the first Server Cleartext packet
-({{packet-server-cleartext}}).  The server MAY choose to use the value that the
-client initially selects.
-
-A server MUST NOT propose a different connection ID in response to a Client
-Cleartext packet ({{packet-client-cleartext}}).  A Client Cleartext packet is
-only sent after the server has committed to maintaining connection state.
+packets ({{packet-client-initial}}) and 0-RTT packets ({{packet-protected}}).
+If the client has received any packet from the server, it uses the connection ID
+it received from the server.
```

Comment:
This sentence is slightly misleading, since 0-RTT packets will continue using the client Connection ID. I would remove this sentence since paras below cover everything else.